src/tools/pub/cmd/pub-global.md

```diff
@@ -91,8 +88,9 @@ package again.
 ## Running a script
 
 You can run a script from an activated package explicitly using
-`pub global run`, or you can add it to your PATH so that you can run it
-directly at the command line.
+`pub global run`. If the script has been added to the
+`executables` tag in its pubspec file, you can also add it to your
+PATH to run it directly at the command line.
```

Comment:
"add it to your path" is a little confusing; you wouldn't add the specific script to your path,  but rather the bin directory in the cache.